cmd/meta.go

```diff
@@ -12,11 +12,14 @@ import (
 func metaCommand() *cobra.Command {
 	metaCmd := &cobra.Command{
 		Use:   "meta <path>",
-		Short: "Prints the metadata of a resource",
+		Short: "Prints the entry's metadata",
+		Long:  `Prints the entry's metadata. By default, meta prints the full metadata as returned by the
```

Comment:
I feel like this description still requires an understanding of plugin design. I'd say something like `Specify the --attribute flag to print the meta attribute - a reduced set of metadata returned when entries are enumerated, also viewable with the 'info' command - instead.`

Reply:
That's much better. I did remove the reference to `info` because it doesn't clarify anything. May be worth removing the `info` command altogether since I originally introduced it to print the `meta` attribute.